OxfordBodleian/Juel-Jensen/BDLaethg28.xml

```diff
@@ -74,45 +81,48 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     <measure unit="leaf">16</measure>
-                                    <measure unit="leaf" type="blank"></measure>
-                                    <!--<locus target=""/>-->
                                     <dimensions type="outer" unit="mm">
                                         <height>105</height>
+
                                         <width>75</width>
                                         <depth>23</depth>
+
                                     </dimensions>
                                     <dimensions type="folio" unit="mm">
                                         <height>103</height>
                                         <width>72</width>
                                     </dimensions>
-                                    <note corresp="#leafdim">Data on leaves dimensions taken from <locus target="#9r"/>.</note>
+                                    <note corresp="#leafdim">Data on leaves dimensions taken from
+                                            <locus target="#9r"/>.</note>
                                     <measure type="weight" unit="g">84</measure>
                                 </extent>
-                                <foliation>Pencil foliation in the upper right corner carried out by the Bodleian Library during our stay 
-                                in January 2020.</foliation>
+                                <foliation>Pencil foliation in the upper right corner.</foliation>
                                 <collation>
                                     <list>
                                         <item xml:id="q1" n="A">
                                             <dim unit="leaf">1</dim>
-                                            <locus from="1r" to="1v"/>
-                                            1, stub before 1
-                                        </item>
+                                            <locus from="1r" to="1v"/> 1, stub before 1 </item>
                                         <item xml:id="q2" n="1">
                                             <dim unit="leaf">7</dim>
-                                            <locus from="2r" to="8v"/>
-                                            6, stub after 1
-                                        </item>
+                                            <locus from="2r" to="8v"/> 6, stub after 1 </item>
                                         <item xml:id="q3" n="2">
                                             <dim unit="leaf">8</dim>
                                             <locus from="9r" to="16v"/>
                                         </item>
                                     </list>
                                 </collation>
+
+                                <condition key="deficient">The binding is broken. A brown liquid
+                                    (coffee?) was spilled on the manuscript, see especially on <locus
```

Comment:
I suspect that some liquid was spilled on the manuscript but that the brown color is actually the ink (perhaps a mixture of black and red) used to draw the doodle on f. 16r and the additional notes here and there (ff. 9r-12r). What do you think?